src/openzaak/components/catalogi/admin/mixins.py

```diff
@@ -135,47 +133,49 @@ def get_actions(self, request):
     publish_selected.short_description = _("Publish selected objects")
 
 
-class NewVersionMixin(object):
-    exclude_copy_relation = []
-
-    def create_new_version(self, obj):
-        old_pk = obj.pk
+class SideEffectsMixin:
```

Comment:
meh, let's tackle that if we need it rather than building something we could maybe possibly re-use in the future. It's defined/scoped in the `catalogi.admin` package for a reason